src/checkers/inference/DefaultSlotManager.java

```diff
@@ -37,22 +39,61 @@
     private final AnnotationMirror unqualified;
     private final AnnotationMirror varAnnot;
 
-    //Whether or not a call to getSlot on a real annotation mirror should generate
-    //a new AnnotationMirror each time or test whether or not we already have the
-    //given annotation and pull it from a store.
-    //This should only be used when annotations are NOT parameterized
-    //TODO: If we wrapped all annotations used by the framework in a special
-    //TODO: smart AnnotationMirror interface that has a useful equals
-    //TODO: We could instead create an LRU for the cases of parameterized annotations
-    private final boolean storeConstants;
-    private final Map<String, ConstantSlot> constantStore;
-
     //this id starts at 1 because sin ome serializer's (CnfSerializer) 0 is used as line delimiters
     //monotonically increasing id for all VariableSlots (including subtypes of VariableSlots)
     private int nextId = 1;
 
-    //a map of variable id to variable for ALL variables (including subtypes of VariableSlots)
+    /**
+     * A map of Integer to VariableSlot for storing all the slots encountered by
+     * this slot manager. If a slot is subtype of another slot, the subtype is
+     * also stored. ConstantSlots are also stored in this map, since
+     * ConstantSlot is subclass of VariableSlot. VariableSlot needs to be down
+     * cast to a subclass such as CombVariableSlot in order to get the concrete
+     * slot type
+     *
+     * @key id of slot used to locate a VariableSlot
```

Comment:
`@key` and `@value` would be useful javadoc tags for documenting maps, but I don't think they exist.
Can you try running javadoc on the code and see if this works?
If we don't have a `gradle javadoc` target yet, could you add one?